File: client/client-app/src/app/_models/medicalimage.ts
export interface ImageResponse {
    image: [][][];
}

export interface MedicalImage {
    id: string | null;
    imageString: string | null;
}

export interface ImageMeta {
    id: number;
    url: string;
    taken: Date;
    patientId: number;
    patient: {
        id: number;
        dateOfBirth: Date;
        created: Date;
        gender: string;
        address: string;
        city: string;
        ssn: string;
        firstName: string;
        middlename: string;
        lastName: string;
        email: string;
        phoneNumber: string;
        extraInfo: null;
    };
}

export interface ImageText {
    text: string;
}

File: final/UITT/API/Entities/MedicalImage.cs
using System;

namespace API.Entities
{
    public class MedicalImage
    {
        public int Id { get; set; }
        public string Url { get; set; }
        public Patient Patient { get; set; }
        public int PatientId { get; set; }      
        public DateTime Taken { get; set; }
    }
}